### PR TITLE
Handle multiple backtesting formats

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: trading/exchanges/exchange_manager.py [hunk 1]
```diff
@@ -172,6 +172,9 @@ def _set_config_time_frame(self):
         if client_shortest_time_frame not in self.time_frames:
             self.time_frames.append(client_shortest_time_frame)
 
+        self.time_frames = TimeFrameManager.sort_time_frames(self.time_frames)
+        self.time_frames.reverse()
```
Comment (Werdor) [inline trading/exchanges/exchange_manager.py]: I think we can simplify here with the [`reverse=True` ](https://www.programiz.com/python-programming/methods/built-in/sorted) arg of `sorted()`.

File: trading/trader/modes/abstract_mode_creator.py [hunk 2]
```diff
@@ -273,10 +273,11 @@ def check_and_adapt_order_details_if_necessary(quantity, price, symbol_market, f
...
         last_prices = await exchange.execute_request_with_retry(exchange.get_recent_trades(symbol))
 
-        reference_sum = sum([float(last_price["price"])
-                             for last_price in last_prices[-ORDER_CREATION_LAST_TRADES_TO_USE:]])
+        used_last_prices = last_prices[-ORDER_CREATION_LAST_TRADES_TO_USE:]
 
-        reference = reference_sum / ORDER_CREATION_LAST_TRADES_TO_USE
+        reference_sum = sum([float(last_price["price"]) for last_price in used_last_prices])
+
+        reference = reference_sum / len(used_last_prices)
```
Comment (Werdor) [inline trading/trader/modes/abstract_mode_creator.py]: Nice !

**Comment by Werdor** [comment]
Really nice job @GuillaumeDSM !

File: trading/exchanges/exchange_manager.py (hunk 1 above)
Comment (GuillaumeDSM) [reply trading/exchanges/exchange_manager.py]: Sure, was not implemented in TimeFrameManager.sort_time_frames, I added it, better like this 👍

**Pull request merged**